apps/health_campaign_field_worker_app/lib/pages/boundary_selection.dart

```diff
@@ -156,11 +156,11 @@ class _BoundarySelectionPageState
                                         // Call the resetChildDropdowns function when a parent dropdown is selected
                                         resetChildDropdowns(label, state);
                                       },
-                                      isRequired: true,
-                                      validationMessage:
-                                          localizations.translate(
-                                        i18.common.corecommonRequired,
-                                      ),
+                                      //isRequired: true,
+                                      // validationMessage:
+                                      //     localizations.translate(
+                                      //   i18.common.corecommonRequired,
+                                      // ),
                                       emptyText: localizations
```

Comment:
"The boundary hierarchy must only be mandatory for the highest boundary". This removes validation for all levels of boundary right??